plugins/inputs/diskio/README.md

```diff
@@ -15,7 +15,6 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 ```toml @sample.conf
 # Read metrics about disk IO by device
-# This plugin ONLY supports Linux
```

Comment:
Should this be present in the sample config? Shouldn't this be outlined in the readme text itself?

More as general remark, as it seems it is done like this for other plugins as well..

Reply:
I removed this line because it is not true. Plugin supports other OSes (see PR description). This line was also removed from `sample.conf`.

Reply:
Oh damn, I completely misinterpreted that 😂